src/components/TitleAndMetaTags/TitleAndMetaTags.js

```diff
@@ -16,6 +19,27 @@ type Props = {
   canonicalUrl: string,
 };
 
+// only provide alternate links to pages in languages where 95-100% of core content has been translated
+// which is determined by status enum of 2
+const completeLanguages = languages.filter(language => {
+  return language.status == 2;
+});
+
+const alternatePages = canonicalUrl => {
+  return completeLanguages.map(language => (
+    <link
+      rel="alternate"
+      hreflang={language.code === 'en' ? 'x-default' : language.code}
```

Comment:
Considering the subdomains, is this the best way?

```suggestion
      hreflang={language.code === window.location.host.split('.')[0] ? 'x-default' : language.code}
```

Reply:
My understanding is that `x-default` is a global definition and shouldn’t be changed due to locale context. It’s treated by Google as the default version of the site if a site doesn’t exist in a certain language or they are unable to detect a user’s language. https://www.hreflangbuilder.com/complete-guide-to-using-the-using-x-default-hreflang-element/

Reply:
@M0nica thanks, I get it, mistakenly thought that in the case of subdomains, we need to mark it as `x-default`.

However, my first question worries me, look at the code of https://www.mozilla.org/en-US/
You see, as I suggested earlier to do two tags - `en` and `x-default` with the same values, I have seen similar on other sites. That is, `x-default` does **not** serve as a replacement for the original language (en).

![image](https://user-images.githubusercontent.com/4408379/58439678-fbac1f80-80dd-11e9-8111-fe3fa1cbcb66.png)

Reply:
Ok, I updated to have all languages that are 100% translated incl. English show up as alternate with their hreflang. And then will also set the `default-x` page as the English version of the page separately. Is that what you had in mind? It seems aligned with Mozilla now.